lib/putasset.js

```diff
@@ -9,14 +9,22 @@ const github = require('@octokit/rest')();
 const mime = require('mime-types');
 const tryToCatch = require('try-to-catch');
 
-module.exports = async (token, {owner, repo, tag, filename}) => {
+module.exports = async (token, {owner, repo, tag, filename, force}) => {
     check(token, {owner, repo, tag, filename});
     
     const type = 'oauth';
     github.authenticate({
         type,
         token,
     });
+
+    await deleteReleaseAssetIfExist({
```

Comment:
I think we can call `deleteReleaseAssetIfExist` only if `force` is set:

```js
if (force)
    await deleteReleaseAssetIfExist();
```

And it is better to do it only in case when we did upload, and error occurred.

Reply:
Yes I admit it would be cleaner and save an unnecessary API call  but I did it that way so we can display a custom error message to tell user why it failed if there is already an asset with the same name and that --force can be used to overwrite this file. 

However I think I can get around it let me check.